RtlsAnchorData[] anchors
geometry_msgs/Point position
bool is_valid_position
